coderd/coderd.go

```diff
@@ -1008,10 +1008,10 @@ func New(options *Options) *API {
 		r.Route("/tasks", func(r chi.Router) {
 			r.Use(apiRateLimiter)
 
+			r.Get("/", api.tasksList)
+
 			r.Route("/{user}", func(r chi.Router) {
 				r.Use(httpmw.ExtractOrganizationMembersParam(options.Database, api.HTTPAuth.Authorize))
-
-				r.Get("/", api.tasksList)
```

Comment:
Review: This path was not conducive to listing other members tasks (e.g. owner).